.github/semantic.yml

```diff
@@ -10,3 +10,5 @@ types:
   - docs
   - doc
   - release
+  - RELEASE
+  - Fix
```

Comment:
Remove this, let's stick to small case only

Reply:
Done